src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+// Actions
+const CREATE = 'CREATE';
+const REMOVE = 'REMOVE';
```

Comment:
- [ ] Hi @SergSkol, great job defining CREATE, and REMOVE action types :+1: Now let's kindly make sure that the value of each action type('CREATE') follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux)(Ex: `const CREATE = bookstore/books/CREATE`)

Reply:
Thanks! It's done